components/spocos.js

```diff
@@ -0,0 +1,330 @@
+import styled from 'styled-components';
+import React, { useState, useEffect } from 'react';
+import Card, {
+  CardHeader,
+  CardTitle,
+  CardContent,
+  CardButtonContainer,
+  CardDiv,
+  TableDiv,
+} from './card';
+import Button from './button';
+import { EDIT, NEW, VIEW, DELETE, COLOR, SPONSORSHIP } from '../constants';
+import {
+  getSponsors,
+  deleteSponsorImagefromStorage,
+  deleteSponsor,
+  updateSponsor,
+  uploadSponsorImageToStorage,
+} from '../utility/firebase';
+import { useAuth } from '../utility/auth';
+import Modal, {
+  Label,
+  ModalContent,
+  ModalField,
+  LogoImage,
+  UploadContainer,
+} from './modal';
+
+// TODO: move font family higher, reduce need to redefine
+
+const Text = styled.p`
+  padding-right: 80px;
+  flex: 1;
+  font-size: 16px;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
```

Comment:
I think according to the design file, the font color is #5A5A5A which is `BODY_TEXT`
![Screen Shot 2020-09-11 at 11 48 04 AM](https://user-images.githubusercontent.com/38872354/92961755-a99f2280-f424-11ea-9b1c-674a520be522.jpg)